chipsec/utilcmd/interrupts_cmd.py

```diff
@@ -24,22 +24,29 @@
 
 from chipsec.command        import BaseCommand
 from chipsec.hal.interrupts import Interrupts
+import uuid
+
 
 # ###################################################################
 #
 # CPU Interrupts
 #
 # ###################################################################
+
+
+
 class SMICommand(BaseCommand):
     """
     >>> chipsec_util smi count
     >>> chipsec_util smi <thread_id> <SMI_code> <SMI_data> [RAX] [RBX] [RCX] [RDX] [RSI] [RDI]
+    >>> chipsec_util smi smmc <RT_code_start> <RT_code_end> <GUID> <payload_loc> <payload_file>
 
     Examples:
 
     >>> chipsec_util smi count
     >>> chipsec_util smi 0x0 0xDE 0x0
     >>> chipsec_util smi 0x0 0xDE 0x0 0xAAAAAAAAAAAAAAAA ..
+    >>> chipsec_util.py smi smmc 0x79dfe000 0x79efdfff ed32d533-99e6-4209-9cc02d72cdd998a7 0x79dfaaaa payload.bin
```

Comment:
Why does the payload need to be put in a file?  Could it also be passed in via cmd line argument?

Reply:
I can make it string or file. Will add